src/components/legend/legend-styles.module.scss

```diff
@@ -1,37 +1,107 @@
+@import 'styles/settings';
```

Comment:
I think this file needs refactoring, but I assume it'll happen anyway because this is the first iteration.